bfdd/control.c

```diff
@@ -435,6 +435,15 @@ static void control_read(struct thread *t)
 		return;
 	}
 
+#define FRR_BFD_MAXLEN 10 * 1024
```

Comment:
Just for my own knowledge: Where is this number come from?

Reply:
The contents of these messages are JSON text which can be big, but in the input case I think it is reasonable to limit the ceiling of the message size to something less than maximum of `uint32_t`. I don't know what value to recommend, because on startup it could be possible someone wants to configure hundreds or thousand of peers (unlikely).

We could also check the message type before `read()`ing so we avoid the allocation at all.

Also this control source will probably go away with YANG/northbound and should be deprecated so people don't start using it.

Reply:
I just made the number up, frankly.
But I'm not sure what to do with Rafael's comment: we have a problem, and it seems like we should do something about it. If it would make sense to accept ... 100KB instead, then fair enough. But I think we need to pick some number and close this hole?